kyuubi-server/src/main/scala/org/apache/kyuubi/server/metadata/jdbc/JdbcDatabaseDialect.scala

```diff
@@ -19,6 +19,10 @@ package org.apache.kyuubi.server.metadata.jdbc
 
 trait JdbcDatabaseDialect {
   def limitClause(limit: Int, offset: Int): String
+  def insertOrReplace(
+      table: String,
+      cols: Seq[String],
+      keyCol: String): Option[String] = None
```

Comment:
can we have a default implementation instead of returning `Option[String]`? Or simply leave empty default implementation as all supported RDBMSs have dedicated dialects

Reply:
oh, I see the issue, it might require more than one SQL to complete the logic

Reply:
Updated.